docs/private-networks/reference/api/index.md

```diff
@@ -2075,6 +2075,68 @@ curl -X POST --data '{"jsonrpc": "2.0","method": "priv_newFilter","params": ["4r
 }
 ```
 
+</TabItem>
+</Tabs>
+
+### `priv_traceTransaction`
+
+Provides a [transaction trace](../../../public-networks/reference/api#trace_transaction) for a private transaction. 
+
+#### Parameters
+- `transactionHash`: _string_ - the hash of the private transaction to trace.
+- `privacyGroupId`: _string_ - the privacy group ID associated with the transaction
```

Comment:
1st param - privacyGroupId
2nd param - transactionHash

Reply:
Hi @NickSneo. I reordered. Could you please review again?